sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -165,8 +168,12 @@ export interface QueueOptions {
   duplicateDetectionHistoryTimeWindow?: string;
 
   /**
-   * The URL of Service Bus queue to forward deadlettered messages to.
-   *
+   * Absolute URI or relative path of the messaging entity the dead-lettered messages are to be forwarded to.
+   * For e.g., Valid relative paths used in the ATOM based management API are
+   *     - `<queue-name>`
+   *     - `<topic-name>`
+   *     - `<topic-name>/Subscriptions/<subscription-name>`
+   *     - `<topic-name>/Subscriptions/<subscription-name>/Rules/<rule-name>`
```

Comment:
Based on the [documentation on auto-forwarding](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-auto-forwarding), one can forward messages to either a queue or a topic. If so, then the relative paths of the messing entity is the same as the name of the entity and there is scope to simplify the docs above

Reply:
updated